gapic/src/main/com/google/gapid/models/Devices.java

```diff
@@ -455,6 +463,13 @@ public String toString() {
       }
     }
 
+    public String errorMessage() {
+      if (internalErr != null) {
+        return "Internal Error: " + internalErr.getErrInternal().getMessage();
```

Comment:
This assumes we don't ever get any other type of error. As I had pointed out in the previous PR, this should be handled by the `Client` class, which already knows how to translate `Service.Error` into messages (via exceptions).